{"uuid": "794abf40-536b-4562-bfd0-86d972a5124b", "children": ["781ed3bb-cd3f-4bfc-b34f-4f87f9be36fa"], "befores": [{"name": "_xunit_setup_method_fixture_TestLogin", "status": "broken", "statusDetails": {"message": "urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))\n", "trace": "  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/_pytest/python.py\", line 821, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/_pytest/python.py\", line 717, in _call_with_optional_argument\n    func()\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/Tests/test_login.py\", line 11, in setup_method\n    self.driver.get(BASE_URL)\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 363, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 352, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py\", line 306, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py\", line 326, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/urllib3/_request_methods.py\", line 143, in request\n    return self.request_encode_body(\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/urllib3/_request_methods.py\", line 278, in request_encode_body\n    return self.urlopen(method, url, **extra_kw)\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/urllib3/poolmanager.py\", line 443, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/urllib3/connectionpool.py\", line 843, in urlopen\n    retries = retries.increment(\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/urllib3/util/retry.py\", line 474, in increment\n    raise reraise(type(error), error, _stacktrace)\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/urllib3/util/util.py\", line 38, in reraise\n    raise value.with_traceback(tb)\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/urllib3/connectionpool.py\", line 789, in urlopen\n    response = self._make_request(\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/urllib3/connectionpool.py\", line 536, in _make_request\n    response = conn.getresponse()\n  File \"/Users/ugurakyay/Desktop/easyPoint_WEB_Automation/Admin_Panel_Automation/.venv/lib/python3.9/site-packages/urllib3/connection.py\", line 507, in getresponse\n    httplib_response = super().getresponse()\n  File \"/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/http/client.py\", line 1349, in getresponse\n    response.begin()\n  File \"/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/http/client.py\", line 316, in begin\n    version, status, reason = self._read_status()\n  File \"/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/http/client.py\", line 285, in _read_status\n    raise RemoteDisconnected(\"Remote end closed connection without\"\n"}, "start": 1726740554768, "stop": 1726740613433}], "start": 1726740554768, "stop": 1726740613564}